Framework/Kernel/test/TimeSeriesPropertyTest.h

```diff
@@ -569,11 +569,11 @@ class TimeSeriesPropertyTest : public CxxTest::TestSuite {
     std::vector<double> values{1., 2., 3.};
     std::vector<double> values_expected{values[0], values[1]};
 
-    auto log_input = std::make_shared<TimeSeriesProperty<double>>("three_values", times, values);
-    auto log_expected = std::make_shared<TimeSeriesProperty<double>>("three_values", times_expected, values_expected);
-    auto log_result_base = std::shared_ptr<Property>(log_input->cloneInTimeROI(roi));
-    auto log_result = std::static_pointer_cast<TimeSeriesProperty<double>>(log_result_base);
-    TS_ASSERT_EQUALS(*log_result, *log_expected);
+    auto tsp_input = std::make_unique<TimeSeriesProperty<double>>("three_values", times, values);
+    auto tsp_expected = std::make_unique<TimeSeriesProperty<double>>("three_values", times_expected, values_expected);
+    auto tsp_result_base = std::shared_ptr<Property>(tsp_input->cloneInTimeROI(roi));
+    auto tsp_result = std::static_pointer_cast<TimeSeriesProperty<double>>(tsp_result_base);
+    TS_ASSERT_EQUALS(*tsp_result, *tsp_expected);
```

Comment:
Please add a check that `tsp_result` isn't equal to `tsp_input` so we know the input data isn't modified

Reply:
Done.